src/themes/default/index.js

```diff
@@ -1,16 +1,9 @@
 import components from './components'
-import {
-  palette,
-  scales,
-  tokens as ogTokens
-} from './deprecated/foundational-styles'
+import { tokens as ogTokens } from './deprecated/foundational-styles'
 
 import tokens from './tokens'
 
 export default {
-  /* DEPRECATED */
-  palette,
-  scales,
   tokens: ogTokens,
```

Comment:
Wondering if we can move off `tokens` throughout this theme since they are all defined at the root level of the theme object?

Reply:
There's one last component that uses `theme.tokens.{}`, so will wait for that to land before this.